.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,11 @@
-### Please check if the PR fulfills these requirements:
-
-* [ ] Tests for the changes have been added/updated (if possible)
-* [ ] Documentation has been updated/added if relevant
-* [ ] Screenshots are attached to Github PR if visual/UI changes were made
-
-### What is the current behavior?
+### Why is this change necessary? How to reproduce the issue?
 <!-- List issue if it fixes/closes/implements one using the "Fixes #<number>" or "Closes #<number>" syntax -->
 
 ### What is the new behavior?
+<!-- Include a screenshot/video if UI changed -->
+
+### Unit Tests and E2E Tests
+<!-- Include Unit Tests and E2E Tests as necessary, or make issue to do so -->
```

Comment:
What are you supposed to write under this section? Regardless if I'm writing an MR with or without tests, I feel like I wouldn't be writing anything under that.